node/storage-cleaner/src/appwrite.js

```diff
@@ -0,0 +1,39 @@
+import { Client, Storage, Query } from 'node-appwrite';
+import { getExpiryDate } from './util.js';
+
+class AppwriteService {
+  constructor() {
+    const client = new Client()
+      .setEndpoint(
+        process.env.APPWRITE_ENDPOINT ?? 'https://cloud.appwrite.io/v1'
+      )
+      .setProject(process.env.APPWRITE_FUNCTION_PROJECT_ID)
+      .setKey(process.env.APPWRITE_API_KEY);
+    this.storage = new Storage(client);
+  }
+
+  /**
+   * Clean up files from the storage bucket by removing files older than a specified retention period.
+   *
+   * @param {string} bucketId - The ID of the storage bucket to clean.
+   * @returns {Promise<void>} A Promise that resolves when the bucket is cleaned.
+   */
+  async cleanBucket(bucketId) {
+    let hasNextPage = true;
+    const queries = [
+      Query.lessThan('$createdAt', getExpiryDate()),
+      Query.limit(100),
```

Comment:
Can we do smaller batches so we don't risk running out of memory/spam the Storage API unnecessarily? 

@Meldiron thoughts?

Reply:
we can change it to 25 instead of 100

Reply:
I have updated it